cypress.config.js

```diff
@@ -12,7 +12,7 @@ const defaultCommandTimeout = process.env.NODE_ENV === "develop" ? 10000 : 4000;
 module.exports = defineConfig({
     e2e: {
         baseUrl: process.env.CYPRESS_MAGENTO2_BASE_URL || envConfig.MAGENTO2_BASE_URL || baseUrl,
-        specPattern: process.env.CYPRESS_MAGENTO2_SPEC_PATTERN || envConfig.MAGENTO2_SPEC_PATTERN ||'cypress/integration/hyva/**/*.spec.js',
+        specPattern: process.env.CYPRESS_MAGENTO2_SPEC_PATTERN || envConfig.MAGENTO2_SPEC_PATTERN ||'cypress/integration/**/*.spec.js',
         excludeSpecPattern: process.env.CYPRESS_MAGENTO2_EXCLUDE_PATTERN || envConfig.MAGENTO2_EXCLUDE_PATTERN || '',
```

Comment:
This default value doesn't make any sense, since it causes both the luma and the hyva tests to be executed.

Reply:
See https://github.com/elgentos/magento2-cypress-testing-suite/issues/65#issuecomment-1223687378